Library/Homebrew/os/mac/sdk.rb

```diff
@@ -0,0 +1,57 @@
+require "os/mac/version"
+
+module OS
+  module Mac
+    class SDK
+      attr_reader :version, :path
+
+      def initialize version, path
+        @version = OS::Mac::Version.new version
+        @path = Pathname(path)
```

Comment:
Very few places in Homebrew core use `Pathname(<string>)` and the majority is using `Pathname.new(<string>)`. Just wanted to point that out. Feel free to ignore this comment, if you deem it irrelevant.